Formula/brew-php-switcher.rb

```diff
@@ -0,0 +1,27 @@
+class BrewPhpSwitcher < Formula
+  desc "Switch Apache / Valet / CLI configs between PHP versions"
+  homepage "https://github.com/philcook/php-switcher"
+  url "https://github.com/philcook/brew-php-switcher/archive/v2.0.tar.gz"
+  sha256 "c2303b1b1a66ee90ed900c3beabacd6aa4e921dbcad5242e399c45d86899bc88"
+  head "https://github.com/philcook/brew-php-switcher.git"
+
```

Comment:
can you add a `bottle :unneeded`? I don't think we need bottles for a single script.

Reply:
Sure. Will update when I get a chance.

Reply:
Added.